modules/core/src/scenegraph/gltf-instantiator/gltf-instantiator.js

```diff
@@ -39,6 +39,22 @@ const fs = `
   }
 `;
 
+const DEFAULT_ATTRIBUTES = {
+  POSITION: {constant: true, value: [0, 0, 0]},
+  NORMAL: {constant: true, value: [0, 0, 0]},
+  TEXCOORD_0: {constant: true, value: [0, 0]}
+};
+
+const ATTRIBUTE_TYPE_TO_COMPONENTS = {
```

Comment:
We have this map in several places. We could export it from loaders.gl/gltf.

That said, I think it is done (or should be done) in GLTFParser, and I think glTFParser.resolve should pre-decode this for the application's use.

At minimum, add a TODO to remove it?